### DISCUSSION TITLE
Supported Grids Besides CONUS

### DISCUSSION THREAD

**Discussion by DavidHuber-NOAA** [ufs-community/ufs-srweather-app#567] [Q&A] [answered]
In [section 5.2](https://ufs-srweather-app.readthedocs.io/en/release-public-v2.1.0/RunSRW.html#grid-configuration) of the docs, 4 predefined and supported grids are listed.  However, in the predef_grid_params.yaml file, several other grids have been created.  Are all of the grids in predef_grid_params.yaml available for use?  In particular, I would like to use Alaska, Hawaii, and Puerto Rico (EMC_AK, EMC_HI, EMC_PR).

**Comment by ShawnCebulaNOAA** [comment, marked as answer]
From what I understand, you should be able to use the listed grids in the YAML file, but the four listed in the support documents are the ones we are able to easily support, so any issues you may run into with grids other than those may not be quickly resolved.  @mark-a-potts @gspetro-NOAA If you could verify that this is the case?  

Thanks David!

Shawn Cebula, EPIC End User Support

**Reply by gspetro-NOAA** [reply]
That is correct!

**Reply by ShawnCebulaNOAA** [reply]
Thanks for the confirm Gillian, and hope this helps David!
